### ISSUE TITLE
Ability to view talent grids/stats for weapons and armor

### ISSUE THREAD

**Issue by BlueKirby** [joshhunt/destinyPlumbing#2]
I notice at the moment that the weapon and armor streams are fairly top-level, with no data on stats and perk combinations. This would definitely be an awesome thing to have in some capacity, even if to begin with the relevant hashes are provided and additional querying can be done.

Is this something you're working on currently, or are you looking for help?

Amazing work so far!

**Comment by joshhunt** [comment]
You should be able to find this image in the full item dumps. Take a look at [DestinyInventoryItemDefinition.json](http://destiny.plumbing/raw/mobileWorldContent/en/DestinyInventoryItemDefinition.json), [DestinyStatDefinition.json](http://destiny.plumbing/raw/mobileWorldContent/en/DestinyStatDefinition.json), and [DestinyStatGroupDefinition.json](http://destiny.plumbing/raw/mobileWorldContent/en/DestinyStatGroupDefinition.json).

I thought about adding stats to the post processed feeds, but even if I just included the hashes, that would nearly double the size of the file.

Is there a minimal set of data that would be helpful but not inflate the file size too much? I might take a look at this over the weekend if I find time.

**Comment by BlueKirby** [comment]
Yeah I see what you mean, I looked through the raw files last night and the data is really extensive. Perhaps it might be worth being able to query an individual weapon (or subset?) and have the full data for them shown that way instead.

I can't see a way to include all the meaningful data that won't bloat file size otherwise. However, on second thought, the main file isn't where it belongs anyway, since the primary use for that is likely just to create a big list.
